cmd/dockerd/daemon.go

```diff
@@ -117,17 +123,18 @@ func (cli *DaemonCli) start(opts daemonOptions) (err error) {
 
 	opts.common.SetDefaultOptions(opts.flags)
 
-	if opts.common.TrustKey == "" {
-		opts.common.TrustKey = filepath.Join(
-			getDaemonConfDir(),
-			cliflags.DefaultTrustKeyFile)
-	}
```

Comment:
I'd have to debug it again, can't immediately see it. But I know it was the case when creating the commit back at the weekend.